modules/ui/sections/background_list.js

```diff
@@ -164,14 +97,41 @@ export function uiSectionBackgroundList(context) {
         });
     }
 
-    function drawListItems(layerList, type, change, filter) {
-        var sources = context.background()
+    function categoryMapping(layer) {
+        var categoryWeights = {
+            'photo': categoryGroups.photo,
+            'historicphoto': categoryGroups.photo,
+            'map': categoryGroups.map,
+            'historicmap': categoryGroups.map,
+            'elevation': categoryGroups.map,
+            'osmbasedmap': categoryGroups.osm,
+            'qa': categoryGroups.osm,
```

Comment:
Is `qa` supposed to be only for OSM-based maps? In the U.S., the TIGER Roads 2020 overlay is also categorized as `qa`, even though it’s purely based on federal census data. This is one of the most important layers in the U.S., the main one used by mappers to name streets, so it would be unfortunate to bury it in the middle of such a long “Maps based on OpenStreetMap” category.

Reply:
I've added (890d654fe61bdcdabd6e987398dd518e05fbbbd5) a separate group for the QA layers. While they are by definition always based on OSM data (aren't they?), they are indeed not always solely based on OSM data. I agree that given their importance it would be a shame to "bury" them in the long list of regular OSM based maps.

However, I've chosen to show the qa section collapsed by default, since the usage and interpretation of these layers is, general speaking, not obvious and could be confusing to use without proper instructions and/or a dedicated map legend which we both don't have at the moment. I think users who want to use the TIGER overlay would likely search it in the "Quality Assurance" section.

Reply:
The TIGER Roads overlay is purely based on census data without any OSM data at all. I haven’t seen a crisp definition of each category anywhere; it could be that the TIGER Roads overlay is simply miscategorized, though mapping based on it could be described as “quality assurance”. I think it’s fine to put it under that section for now. If we hear of people having a hard time finding it, we can always recategorize it as “other”.